pkg/controller/lifecycle/actuator.go

```diff
@@ -74,113 +67,14 @@ func (a *actuator) Reconcile(ctx context.Context, _ logr.Logger, ex *extensionsv
-			reportSuspensionContinuation = pointer.String("on")
-		} else {
-			reportSuspensionContinuation = pointer.String("off")
-		}
-	}
-
-	filters := computeLogFilters(shootRsyslogRelpConfig.LoggingRules)
-
-	rsyslogConfigValues := map[string]interface{}{
-		"target":                       shootRsyslogRelpConfig.Target,
-		"port":                         shootRsyslogRelpConfig.Port,
-		"projectName":                  projectName,
-		"shootName":                    cluster.Shoot.Name,
-		"shootUID":                     cluster.Shoot.UID,
-		"filters":                      filters,
-		"rebindInterval":               shootRsyslogRelpConfig.RebindInterval,
-		"timeout":                      shootRsyslogRelpConfig.Timeout,
-		"resumeRetryCount":             shootRsyslogRelpConfig.ResumeRetryCount,
-		"reportSuspensionContinuation": reportSuspensionContinuation,
-		"metrics": map[string]interface{}{
-			"textfileDir": nodeExporterTextfileCollectorDir,
-		},
-	}
-
-	if shootRsyslogRelpConfig.TLS != nil && shootRsyslogRelpConfig.TLS.Enabled {
-		refSecretName, err := lookupReferencedSecret(cluster, *shootRsyslogRelpConfig.TLS.SecretReferenceName)
-		if err != nil {
-			return err
-		}
-
-		refSecret := &corev1.Secret{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      refSecretName,
-				Namespace: ex.Namespace,
-			},
-		}
-		if err = a.client.Get(ctx, client.ObjectKeyFromObject(refSecret), refSecret); err != nil {
-			return err
-		}
-
-		if err := utils.ValidateRsyslogRelpSecret(refSecret); err != nil {
-			return err
-		}
-
-		var permittedPeers []string
-		for _, permittedPeer := range shootRsyslogRelpConfig.TLS.PermittedPeer {
-			permittedPeers = append(permittedPeers, strconv.Quote(permittedPeer))
-		}
-
-		var authMode string
-		if shootRsyslogRelpConfig.TLS.AuthMode != nil {
-			authMode = string(*shootRsyslogRelpConfig.TLS.AuthMode)
-		}
-
-		var tlsLib string
-		if shootRsyslogRelpConfig.TLS.TLSLib != nil {
-			tlsLib = string(*shootRsyslogRelpConfig.TLS.TLSLib)
-		}
-
-		rsyslogConfigValues["tls"] = map[string]interface{}{
-			"enabled":       shootRsyslogRelpConfig.TLS.Enabled,
-			"permittedPeer": strings.Join(permittedPeers, ","),
-			"authMode":      authMode,
-			"tlsLib":        tlsLib,
-			"ca":            refSecret.Data["ca"],
-			"crt":           refSecret.Data["crt"],
-			"key":           refSecret.Data["key"],
-		}
-	}
-
-	values := map[string]interface{}{
-		"rsyslogConfig": rsyslogConfigValues,
-		"auditdConfig": map[string]interface{}{
-			"enabled": true,
-		},
-		"images": map[string]interface{}{
-			"alpine": imagevector.AlpineImage(),
-			"pause":  imagevector.PauseContainerImage(),
-		},
-		"pspDisabled": gardencorev1beta1helper.IsPSPDisabled(cluster.Shoot),
-	}
-
-	release, err := chartRenderer.RenderEmbeddedFS(charts.InternalChart, charts.RsyslogConfiguratorChartPath, releaseName, metav1.NamespaceSystem, values)
-	if err != nil {
+	// TODO(plkoknanov): remove this after a couple releases.
```

Comment:
L61-L68 seems no longer needed, you can drop it:
https://github.com/gardener/gardener-extension-shoot-rsyslog-relp/blob/b0b214bd593fd46f4fe571b68cc04e9a4edb7d9c/pkg/controller/lifecycle/actuator.go#L61-L68